File: Python Code/Daily_Flash_Code_0/Week9/19Mar/51-DailyFlash_Solutions/19-Mar/C++/prog4.cpp
#include<iostream>

int main() {
	int num = 1;
	for(int i = 0; i < 4; i++) {
		for(int j = 0; j < 7 - i; j++) {
			if(j < i) {
				std::cout << " ";
			} else {
				if(i % 2 == 0) {
					std::cout << num;
					num++;
					if(j == 6 - i)
						num--;
				}

				else {
					std::cout << num;
					num--;
					if(j == 6 - i)
						num++;
				}
			}
		}
		std::cout << "\n";
	}
}

File: Python Code/Daily_Flash_Code_0/Week9/19Mar/51-DailyFlash_Solutions/19-Mar/Python/prog4.py
num = 1
for i in range(4):
    for j in range(7 - i):
        if(j < i):
            print("", end = " ")
        else:
            if(i % 2 == 0):
                print(num, end = "")
                num += 1
                if(j == 6 - i):
                    num -= 1
            else:
                print(num, end = "")
                num -= 1
                if(j == 6 - i):
                    num += 1
    print()
